odl/test/solvers/functional/default_functionals_test.py

```diff
@@ -516,5 +516,61 @@ def test_moreau_envelope_l2_sq(space, sigma):
                             x * 2 / (1 + 2 * sigma))
 
 
+def test_bregman_functional_no_gradient(space):
+    """Test that the Bregman distance functional fails if the underlying
+    functional does not have a gradient and no subgradient operator is
+    given."""
```

Comment:
Move """ to newline

Reply:
Also first line should be "sufficient" and fit in one line, the rest should elaborate